lib/tasks/award_badge_once.rake

```diff
@@ -0,0 +1,30 @@
+namespace :award_badge_once do
+  desc "TODO"
+  task award_organized_badge: :environment do
+        @projects = Project.all
+        @edition = Edition.last
+        @projects.each do |project|
+        	is_organised_flag = true
+          @edition.weeks.each do |week|
```

Comment:
instead of `@edition` use `project.edition`

And more importantly I'd rather use query for that. You can add migration to link `Week` with `Task` model and then do in project loop:

```
results = project.tasks.joins(:week).group("weeks.id").count -> will return hash with week_id and number of tasks  
results.length == 12 -> award badge
```

Reply:
@DMips @pg20 Can you confirm the usecase for this badge once? As per our mapping currently, it is 'Badge awarded when mentee enters roadmap'. I guess it's incorrect.